svelte/src/routes/v2/components/config-changer.svelte

```diff
@@ -1,12 +1,20 @@
-<script>
+<script lang="ts">
 	import Heading from "$ui/heading.svelte";
-
+	import type { ConfigInterface } from "../types";
+    let dependencyLiftTolarance: string;
+    export let config: ConfigInterface;
 </script>
 <div>
     <Heading class="mt-2">
         Config Changer
     </Heading>
     <div>
-        to collapse node, you can click on the red button
+        to collapse node, you can click on the red button <br />
+        to collapse node, click the green button. Tolerance: <input type='input' style='width: 2em; border: 1px solid black; margin: 0 0 10px 4px;' 
```

Comment:
you can use tailwind here instead of style attribute

Reply:
Sorry, don't want to spend time learning tailwind rn. (will get around to that evantually)